packages/mobile/src/account/Education.tsx

```diff
@@ -180,6 +180,7 @@ export default class Education extends React.Component<Props, State> {
             loop={false}
             dotStyle={dotStyle}
             activeDotStyle={activeDotStyle}
+            removeClippedSubviews={false}
```

Comment:
Having this as `true` is a performance improvement for big lists, but this element has only 4 small images and without this the first image is blank until the user scrolls a little. It was the suggested fix on an issue from the library: https://github.com/leecade/react-native-swiper/issues/1199